modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -217,71 +217,68 @@
   (log/info (u/format-color 'blue "[ok]")))
 
 (deftest redshift-types-test
-  (mt/test-driver
-    :redshift
+  (mt/test-driver :redshift
     (testing "Redshift specific types should be synced correctly"
-      (let [db-details   (tx/dbdef->connection-details :redshift nil nil)
-            tbl-nm       "redshift_specific_types"
+      (let [tbl-nm       "redshift_specific_types"
             qual-tbl-nm  (format "\"%s\".\"%s\"" (redshift.test/unique-session-schema) tbl-nm)
             view-nm      "late_binding_view"
-            qual-view-nm (format "\"%s\".\"%s\"" (redshift.test/unique-session-schema) view-nm)]
-        (t2.with-temp/with-temp [Database database {:engine :redshift, :details db-details}]
-          (try
+            qual-view-nm (format "\"%s\".\"%s\"" (redshift.test/unique-session-schema) view-nm)
+            database     (mt/db)]
```

Comment:
We don't try to create a new database, we just use the `test-data` database. This is for speed (syncing an existing database is faster than syncing a new one) but also so we can use its connection pool.